File: app/src/main/java/me/ykrank/s1next/view/adapter/FavouriteRecyclerViewAdapter.java
package me.ykrank.s1next.view.adapter;

import android.app.Activity;

import me.ykrank.s1next.view.adapter.delegate.FavouriteAdapterDelegate;

public final class FavouriteRecyclerViewAdapter extends BaseRecyclerViewAdapter {
    
    public FavouriteRecyclerViewAdapter(Activity activity) {
        super(activity);

        addAdapterDelegate(new FavouriteAdapterDelegate(activity));

        setHasStableIds(true);
    }
}

File: app/src/main/java/me/ykrank/s1next/view/adapter/FavouriteRecyclerViewAdapter.kt
package me.ykrank.s1next.view.adapter

import android.app.Activity
import androidx.lifecycle.LifecycleOwner
import me.ykrank.s1next.view.adapter.delegate.FavouriteAdapterDelegate

class FavouriteRecyclerViewAdapter(
    activity: Activity,
    private val lifecycleOwner: LifecycleOwner,
) : BaseRecyclerViewAdapter(
    activity
) {
    init {
        addAdapterDelegate(FavouriteAdapterDelegate(activity, lifecycleOwner))
    }
}
